webdriverio/webdriver/examples/visual-e2e/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test.visual.sauce": "npx wdio test/configs/wdio.shared.conf.js",
+    "test.visual.crossplatform": "npx wdio test/configs/wdio.cross.platform.conf.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@wdio/cli": "^7.10.1",
+    "@wdio/jasmine-framework": "^7.10.1",
+    "@wdio/local-runner": "^7.10.1",
+    "@wdio/mocha-framework": "^7.10.1",
+    "@wdio/sauce-service": "^7.10.1",
```

Comment:
You are using the `@wdio/sauce-service`, but I'm not seeing it being used in the configs. Is it really needed?

Reply:
@wswebcreation should I be using this service somewhere? Am I doing something wrong?